compiler/test/suites/gc.re

```diff
@@ -81,7 +81,7 @@ describe("garbage collection", ({test, testSkip}) => {
   );
   assertRunGCError(
     "fib_gc_err",
-    1024,
+    512,
```

Comment:
This no longer ran out of memory at 1024 bytes because no closure is allocated when `fib` is called anymore. I adjusted it down.